persp-mode.el

```diff
@@ -2253,6 +2261,13 @@ killed, but just removed from a perspective(s)."
 
 ;; Perspective funcs:
 
+(defun persp-update-persp-names-cache (new-persp-names)
```

Comment:
_**persp**-update-**persp**-names-cache_
In my mind _persp-_ must be used as a prefix only.

Reply:
You are right. How about renaming it to `persp-update-names-cache`?

Reply:
> persp-update-names-cache

Yes, that's it.

Reply:
I have renamed it. Do you have any other suggestions?

Reply:
Old name remains in the commit message.